static/js/settings_ui.js

```diff
@@ -50,17 +47,13 @@ export function do_settings_change(request_method, url, data, status_element, op
                 display_checkmark(spinner);
             }, appear_after);
             if (success_continuation !== undefined) {
-                if (opts !== undefined && opts.success_continuation_arg) {
-                    success_continuation(opts.success_continuation_arg);
-                } else {
-                    success_continuation(reponse_data);
-                }
+                success_continuation(reponse_data);
```

Comment:
I see we never used `success_continuation_arg`

Reply:
Yeah, we’re not writing C, so if one needs an argument there’s no reason not to close it into `success_continuation`.